presto-main/src/test/java/com/facebook/presto/catalogserver/TestCatalogServerResponse.java

```diff
@@ -179,15 +188,16 @@ public void testGetView()
         assertEquals(viewDefinition.getCatalog(), actualViewDefinition.getCatalog());
         assertEquals(viewDefinition.getSchema(), actualViewDefinition.getSchema());
         assertEquals(viewDefinition.getOwner(), actualViewDefinition.getOwner());
+        assertEquals(metadataEntry.getIsCacheHit(), false);
     }
 
     @Test
     public void testGetMaterializedView()
             throws Exception
     {
-        String connectorMaterializedViewDefinitionJson = testingCatalogServerClient.getMaterializedView(null, null, null);
+        CatalogServerClient.MetadataEntry<String> metadataEntry = testingCatalogServerClient.getMaterializedView(null, null, null);
         ConnectorMaterializedViewDefinition connectorMaterializedViewDefinition = objectMapper.readValue(
-                connectorMaterializedViewDefinitionJson,
+                metadataEntry.getReturnValue(),
                 ConnectorMaterializedViewDefinition.class);
         String originalSql = "SELECT\n  name\n, nationkey\nFROM\n  test_customer_base\n";
         String schema = "tpch";
```

Comment:
Actually let's change all `new ArrayList<>(Arrays.asList(...))` to `ImmutableList.of(...)`

Reply:
done